README.markdown

```diff
@@ -296,15 +274,6 @@ func reticulateSplines(spline: [Double]) -> Bool {
 }
 ```
 
-For functions with long signatures, add line breaks at appropriate points and add an extra indent on subsequent lines:
```

Comment:
i actually like this. but only for method names that run beyond a page guide (the guide helps keep lines within small windows)

![page_guide](https://cloud.githubusercontent.com/assets/1334411/7592544/7ed89018-f899-11e4-977f-088b97d88efb.png)